charts/instana-agent/v1.2.1/questions.yml

```diff
@@ -7,7 +7,7 @@ questions:
 # Basic agent configuration
```

Comment:
It might be good to add a question for `cluster.name` as a required value. Installation will fail if you don't specify it, but you can't set it unless you edit it as a YAML.
```
CreateContainerConfigError: couldn't find key cluster_name in ConfigMap instana-agent/instana-agent
```